authors/apps/authentication/backends.py

```diff
@@ -1,9 +1,84 @@
-# import jwt
-#
-# from django.conf import settings
-#
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+import jwt
+
+from django.conf import settings
+
+from rest_framework import authentication, exceptions
+
+from .models import User
+
 
 """Configure JWT Here"""
+
+
+class JWTAuthentication(authentication.BaseAuthentication):
+    auth_header_prefix = 'Bearer'.lower()
```

Comment:
Please add a space after the prefix to avoid concatenation of the prefix and the actual token, I suggest 'Bearer '

Reply:
@BrianSerem I am not concatenating anything. I simply use it for comparison. If I add a space, the validation will fail, because if you look later in the code I run the `split()` function which removes all whitespace anyway. I don't think this change is necessary.